pkg/roachprod/roachprod.go

```diff
@@ -751,7 +761,34 @@ func Start(
 	if err != nil {
 		return err
 	}
-	return c.Start(ctx, l, startOpts)
+	if err = c.Start(ctx, l, startOpts); err != nil {
+		return err
+	}
+	createClusterConfig(ctx, l, c, startOpts)
```

Comment:
This handles the centralized prometheus service. What about the prometheus instances that's running within the same cluster; i.e., optionally started via `roachprod grafana-start`? 

@DarrylWong I seem to recall you did an informal poll to see who's (still) using it?

Reply:
Kinda, I asked around to see who was still using the Grafana dashboard (`cfg.Grafana.Enabled`) since annotations just applied to that, which was just DR and Admission Control.

Actually using the metrics scraped by prom I did not ask about, but it seems irrelevant in this case I think? The instance started by `roachprod grafana-start` still uses a hardcoded port, and is unaffected setting `AdminUIPort` to 0.

https://github.com/cockroachdb/cockroach/blob/8acf7a2121678164b5765d69f923a2f96577c3b3/pkg/roachprod/roachprod.go#L1784

Reply:
Kinda, I asked around to see who was still using the Grafana dashboard (cfg.Grafana.Enabled) since annotations just applied to that, which was just DR and Admission Control.

Actually using the metrics scraped by prom I did not ask about, but it seems separate in this case I think? The instance started by roachprod grafana-start still uses a hardcoded port, and is unaffected setting AdminUIPort to 0.

https://github.com/cockroachdb/cockroach/blob/8a37a60032db5688da140731a90a7cb2a773f230/pkg/roachprod/prometheus/prometheus.go#L209

So looks like we just have to pass the AdminUI port to that, which shouldn't really have to do with this API.